frontend/src/pages/UserProfileConfiguration/UpdateUserData.tsx

```diff
@@ -0,0 +1,31 @@
+import { useState } from 'react';
+import { UserType } from 'pages/UserProfileConfiguration/types';
+import APIService from '../../Utils/Api/APIService';
+import { updateUser } from '../../Utils/Utils';
+
+type UserUpdateProp = {
+  token: string;
+  user: UserType;
+};
+
+export const UpdateUserData = ({ user, token }: UserUpdateProp) => {
+  const [userBio, setUserBio] = useState('');
```

Comment:
is this value will be always empty string?

e.g.: user wants to edit his 500 characters description, in this solution he has to copy all the description and paste into input, then he has to find a typo and then save proper version

for me is not the best from UX, on edit he should have already populated bio, and then edit what he want to